src/solidity-frontend/solidity_convert.cpp

```diff
@@ -2578,6 +2777,32 @@ bool solidity_convertert::get_constructor_call(
 
   const nlohmann::json constructor_ref = find_decl_ref(ref_decl_id);
 
+  // check for implicit constructor
```

Comment:
This could be put in a separate function something like `get_implicit_ctor_call`.